CommanderConda/construct.yaml

```diff
@@ -24,19 +24,19 @@ conda_default_channels:
 specs:
   - bzip2
   - conda {{ version.split("-")[0] }}
-  - dash 1.19
-  - dash-bootstrap-components
+  - dash 2.0.0 
+  - dash-bootstrap-components 1.0.3
   - fasteners
   - ipykernel
   - ipywidgets 7.6.*
-  - jupyter-dash 0.4.*
-  - jupyterlab 3.0.*
+  - jupyter-dash 0.4.1
+  - jupyterlab 3.2.9 
```

Comment:
@PlatinumCD is there anyway to loosen the requirements? I think a specific patch version is probably unneeded, and specifying a minor version could even be excessive, but I'm not 100% sure.